value.go

```diff
@@ -429,36 +427,30 @@ func (v *Value) AsTime() nano.Ts {
 
 func (v *Value) MissingAsNull() *Value {
```

Comment:
Why keep this as a non-pointer? And also perhaps `v.IsMissing()` should get rid of the check for `v` being nil since this isn't a thing any longer?

Reply:
Good catch. No reason for this to return a pointer.

To return a value I'll have to remove the second line here, but that seems fine.

https://github.com/brimdata/zed/blob/60bbaa1706c3596dfbd0fa3f0701280d7c165669/zio/zeekio/parser_test.go#L82-L83

Reply:
Done in 1af46ba90da7bae8bbd61223eaffbcfc5d0eec37.